### DISCUSSION TITLE
laravel query bug

### DISCUSSION THREAD

**Discussion by medbak** [laravel/framework#33243] [General] [answered]
- Laravel Version: Laravel Framework Lumen (7.1.3) (Laravel Components ^7.0)
- PHP Version: PHP 7.3.19-1+ubuntu20.04.1+deb.sury.org+1 (cli)
- Database Driver & Version:

### Description:
I tried to execute this query : 
$email = Email::where('email', false)
    ->first();
normally the result must be null but I get the first line in the table,
even if tried 
$email = Email::where('email', false)
    ->get();
it returns all the values.

**Comment by GrahamCampbell** [comment, marked as answer]
`Email::whereNull('email')->get()`.

**Reply by medbak** [reply]
Thanks,
But what the returns of
$email = Email::where('email', false)
->first();
must be empty because in the database there is no email named false.
Flase is the result of another test, so the result will be passed to the query

**Reply by jonathan-otr** [reply]
**Email::where('email','=','false')->first()**
